samples/cellular/nrf_cloud_rest_fota/CMakeLists.txt

```diff
@@ -12,4 +12,8 @@ zephyr_compile_definitions(PROJECT_NAME=${PROJECT_NAME})
 
 # NORDIC SDK APP START
 target_sources(app PRIVATE src/main.c)
+
+if(CONFIG_NRF_CLOUD_FOTA_SMP AND CONFIG_BOARD_NRF9160DK_NRF9160_NS)
```

Comment:
is this specifically tied to a board or is it just the non-secure core? I.e. how can customers use this on their own boards?

Reply:
for this sample we are only providing an example for this (`nrf9160dk/nrf9160/ns`) build target.
customers would need to modify it to work with other boards.